cdc/owner/schema.go

```diff
@@ -85,11 +86,32 @@ func (s *schemaWrap4Owner) AllPhysicalTables() []model.TableID {
 	return s.allPhysicalTablesCache
 }
 
+// AllTableNames returns the table names of all tables that are being replicated.
+// NOTICE: AllTableNames is not thread-safe.
+func (s *schemaWrap4Owner) AllTableNames() []model.TableName {
+	if s.allTableNamesCache != nil {
+		return s.allTableNamesCache
```

Comment:
Why should we get the tableNames from this cache instead of fetching the latest tables from `s.schemaSnapshot.Tables()`?

Reply:
Because we may not have ddl for a long time, using cache can improve performance.